pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -184,9 +185,12 @@ func New(obj runtime.Object, handle framework.Handle) (framework.Plugin, error)
 }
 
 func (c *CapacityScheduling) EventsToRegister() []framework.ClusterEvent {
+	// To register a custom event, follow the naming convention at:
+	// https://git.k8s.io/kubernetes/pkg/scheduler/eventhandlers.go#L403-L410
+	eqGVK := fmt.Sprintf("elasticquotas.v1alpha1.%v", scheduling.GroupName)
 	return []framework.ClusterEvent{
 		{Resource: framework.Pod, ActionType: framework.Delete},
-		// TODO: once bump the dependency to k8s 1.22, addd custom object events.
+		{Resource: framework.GVK(eqGVK), ActionType: framework.Add | framework.Update},
```

Comment:
We also need to include the Delete event for elasticquota. Pod may be scheduled successfully if the eq is deleted.

Reply:
Can you elaborate in which circumstance a deleted eq would potentially make a pod (failed by this plugin) schedulable? My understanding is that the unschedulable pods needs to be get its own label updated, and it's irrelevant with EQ's change.

Reply:
I wonder if there is a scenario.
Scheduling of PodA in namespace A fails because the used in eq of namespace A will exceed the max rather than a lack of cluster resources. Then we remove the eq in namespace A so that it is no longer restricted to pod in namespace A. At this point we may need to have the pod in namespace A rescheduled.

But in fact we are not allowed to delete eq in a production environment if there are pods in the namespace A, unless the pods are deleted first.

Reply:
sg, I will update the event to `All`.